src/index.ts

```diff
@@ -77,6 +80,15 @@ export default class SwupJsPlugin extends Plugin {
 		}
 
 		this.options = { ...this.defaults, ...options };
+
+    // Make sure we have a fallback animation available
```

Comment:
This can probably be simplified by just returning `foundAnimation || this.defaultAnimation` from the method looking for the best animation match.